spec/helpers/users_helper_spec.rb

```diff
@@ -1,15 +1,35 @@
 require 'spec_helper'
 
 describe UsersHelper do
+  let(:user) { create(:user, email: 'johndoe@example.com') }
+
   describe '#gravatar_for' do
-    it "returns the image tag for the specified user's gravtar image" do
-      user = create(:user, email: 'johndoe@example.com')
-      expect(gravatar_for(user)).to eq('<img alt="John Doe" class="gravatar" src="https://secure.gravatar.com/avatar/fd876f8cd6a58277fc664d47ea10ad19?s=48" />')
+    context 'when gravatar feature is enabled' do
+      before do
+        ROLLOUT.activate(:gravatar)
+      end
+
+      it "returns the image tag for the specified user's gravtar image" do
+        expect(gravatar_for(user)).to eq('<img alt="John Doe" class="gravatar" src="https://secure.gravatar.com/avatar/fd876f8cd6a58277fc664d47ea10ad19?s=48" />')
+      end
+
+      it "returns the image tag for the specified user's gravtar image with size" do
+        expect(gravatar_for(user, size: 128)).to eq('<img alt="John Doe" class="gravatar" src="https://secure.gravatar.com/avatar/fd876f8cd6a58277fc664d47ea10ad19?s=128" />')
+      end
+
+      after do
+        ROLLOUT.deactivate(:gravatar)
+      end
     end
 
-    it "returns the image tag for the specified user's gravtar image with size" do
-      user = create(:user, email: 'johndoe@example.com')
-      expect(gravatar_for(user, size: 128)).to eq('<img alt="John Doe" class="gravatar" src="https://secure.gravatar.com/avatar/fd876f8cd6a58277fc664d47ea10ad19?s=128" />')
+    context 'when gravatar feature is disabled' do
+      before do
+        expect(ROLLOUT.active?(:gravatar)).to eq(false)
```

Comment:
Should we be doing the same thing with Rollout in both of these contexts? Either do the real activation and deactivation or stub the behavior. It looks like they are being treated differently.